web/ui/dashboard/src/app/guards/signup/signup.guard.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+import { LoginService } from 'src/app/public/login/login.service';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class SignupGuard implements CanActivate {
+	constructor(private router: Router, private loginService: LoginService) {}
+
+	canActivate() {
+		const isSignupEnabled = this.loginService.signupConfig && location.hostname !== 'dashboard.getconvoy.io';
```

Comment:
This isn't just for our cloud @Oluwadaminiola

Reply:
In any instance, nobody should be able to use the signup page if signup is disabled in Convoy config.